Mathlib/Order/Basic.lean

```diff
@@ -1189,6 +1189,9 @@ namespace Prod
 instance (α : Type u) (β : Type v) [LE α] [LE β] : LE (α × β) :=
   ⟨fun p q ↦ p.1 ≤ q.1 ∧ p.2 ≤ q.2⟩
 
+instance instDecidableLE (α : Type u) (β : Type v) [LE α] [LE β] (x y : α × β)
```

Comment:
```suggestion
-- porting note: new instance
instance instDecidableLE (α : Type u) (β : Type v) [LE α] [LE β] (x y : α × β)
```
since it sounds like this was added to help with porting.